### PR TITLE
Fix for TNL-3065 and issue #15

### PR DESCRIPTION
**Pull request by e-kolpakov**
* Fix for [TNL-3065](https://openedx.atlassian.net/browse/TNL-3065): poll shows voting options if voted and subsection navigation is used
* Fix for issue #15: reordering answers prematurely submit block with no modifications

### CODE REVIEW

**Comment by e-kolpakov** [comment]
@smarnach please review this one than :)

File: poll/public/js/poll_edit.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+debugger;
```
Comment (smarnach) [inline poll/public/js/poll_edit.js]: Please remove.

Comment (e-kolpakov) [inline poll/public/js/poll_edit.js]: @smarnach done - thank you for catching this

**Comment by smarnach** [comment]
:+1: Looks all good.

**Comment by e-kolpakov** [comment]
@sarina Sorry, I've totally forgot we need an upstream review for this in order to deliver a fix and already merged this PR. Still, it needs a review, as it is fixes TNL-3065. Do you think it's possible to avoid reverting it and address review notes in a separate PR?

**Comment by sarina** [comment]
@e-kolpakov I'll take a look now

**Comment by sarina** [comment]
@e-kolpakov looks reasonable to me. Have you updated the version in edx-platform?

**Comment by e-kolpakov** [comment]
@sarina only on edx-solutions fork. Should I update upstream now?

**Comment by sarina** [comment]
Yeah let's do that

**Comment by e-kolpakov** [comment]
@sarina ok, updating: https://github.com/edx/edx-platform/pull/9403

**Pull request merged**